src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProvidersConfigProducer.java

```diff
@@ -26,10 +26,10 @@ public class EmbeddingProvidersConfigProducer {
   @ApplicationScoped
   @Startup
   @Retry(
-      maxRetries = 1,
+      maxRetries = 30,
```

Comment:
Shouldn't this be in other PR, to avoid bundling unrelated changes?

Reply:
Actually it's fine either way.

Reply:
Yeah, I forget to change it back yesterday.

Reply:
ah.